plugins/redfish/fu-redfish-backend.c

```diff
@@ -16,6 +16,7 @@
 #include "fu-redfish-multipart-device.h"
 #include "fu-redfish-request.h"
 #include "fu-redfish-smbios.h"
+#include "fu-redfish-smc.h"
```

Comment:
`fu-redfish-smc-device.h` perhaps -- or did you forget to `git add` a header?

Reply:
The former